### PR TITLE
Ported the ACE2 Yardage 450 laser rangefinder

### PR DESCRIPTION
**Pull request by ulteq**

### CODE REVIEW

File: addons/yardage450/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="MX2A">
+        <Key ID="STR_ACE_Yardage450_DisplayName">
+            <English>Yardage 450</English>
+        </Key>
```
Comment (gienkov) [inline addons/yardage450/stringtable.xml]: Can you add?
```<Polish>Yardage 450</Polish>```

File: addons/yardage450/stringtable.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <Key ID="STR_ACE_Yardage450_Description">
+            <English>Laser Rangefinder</English>
+            <German>Laserentfernungsmesser</German>
+        </Key>
```
Comment (gienkov) [inline addons/yardage450/stringtable.xml]: ```<Polish>Dalmierz laserowy</Polish>```

File: addons/yardage450/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        <Key ID="STR_ACE_Yardage450_PowerButtonKey">
+            <English>Yardage 450 - Power Button</English>
+            <German>Yardage 450 - Einschalt-Taste</German>
+        </Key>
```
Comment (gienkov) [inline addons/yardage450/stringtable.xml]: ```<Polish>Yardage 450 - Przycisk zasilania</Polish>```

**Comment by VKing6** [comment]
Does it really say "Target Acquired"?

Also how do you trigger it? I'd suggest using the vector range key.

**Comment by ulteq** [comment]
> Does it really say "Target Acquired"?

It does: https://youtu.be/ieOCrxdhmxs?t=30

> Also how do you trigger it?

1. Press the power button once to activate it
2. Keep the power button pressed to fire the laser until the target is acquired.

>  I'd suggest using the vector range key.

It should be the same button by default.

**Comment by VKing6** [comment]
Looks cool.

**Comment by esteldunedain** [comment]
:+1: Looks good

**Pull request merged**